strawberryfields/engine.py

```diff
@@ -609,22 +609,20 @@ def connection(self) -> xcc.Connection:
         )
 
     @property
-    def device_spec(self) -> DeviceSpec:
+    def device(self) -> Device:
         """The specification of the target device.
 
         Returns:
-            DeviceSpec: the device specification
+            ~strawberryfields.Device: the device specification
```

Comment:
```suggestion
            .strawberryfields.Device: the device specification
```

Reply:
Should this docstring be updated to say "device" instead of "device specification"?